AppCenter/AppCenter/MSACAppCenter.h

```diff
@@ -68,76 +69,45 @@
  *
  * @param services Array of services to start.
  */
-+ (void)startFromLibraryWithServices:(NSArray<Class> *)services;
++ (void)startFromLibraryWithServices:(NSArray<Class> *)services NS_SWIFT_NAME(startFromLibrary(services:));
 
 /**
- * Check whether the SDK has already been configured or not.
- *
- * @return YES if configured, NO otherwise.
+ * The flag indicates whether the SDK has already been configured or not.
  */
-+ (BOOL)isConfigured;
+@property(class, atomic, readonly) BOOL isConfigured;
 
 /**
- * Check whether app is running in App Center Test Cloud.
- *
- * @return true if running in App Center Test Cloud, false otherwise.
+ * The flag indicates whether app is running in App Center Test Cloud.
  */
-+ (BOOL)isRunningInAppCenterTestCloud;
+@property(class, atomic, readonly) BOOL isRunningInAppCenterTestCloud;
```

Comment:
ObjC properties should not be prefixed by "is". In ObjC only the getter should be prefixed by "is". Example `@property (assign, getter=isEditable) BOOL editable;` cf [Apple doc](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html). The good new is it won't break ObjC customers :).